documentation/docs/steps/abapEnvironmentCreateSystem.md

```diff
@@ -68,3 +68,64 @@ create-services:
   plan:   "standard"
   parameters: "{ \"admin_email\" : \"user@example.com\", \"description\" : \"ABAP Environment Q System\", \"is_development_allowed\" : true, \"sapsystemname\" : \"H02\", \"size_of_persistence\" : 4, \"size_of_runtime\" : 1 }"
 ```
+
+## Example: Configuration of an abap-oem Service
+
+If you want to use an abap-oem System please note that the following configurations need to be specified correctly:
+
+* `abapSystemParentSaasAppname`: needs to fulfill the syntax [a-zA-Z0-9\-\_]+
+* `abapSystemConsumerTenantLimit`: The number of consumer tenants tmust be greater or equal 1
+* Either `abapSystemParentServiceLabel` or `abapSystemParentSaasAppname` must be set - depending on who created the oem-instance
+
+You can either specify the parameters required for an abap-oem System either directly in the Jenkinsfile or in a `manifest.yml` file.
+
+The first example makes use of the Jenkinsfile.
+
+```yaml
+steps:
```

Comment:
What about the addon? To install the addon, the addon descriptor needs to be provided via addonDescriptorFileName.

Or is this somehow replaced by the PartenService?

Reply:
This is an example that I think should work:

```
Integration Tests:
    cfOrg : 'itapc-partner-saas-build-test-staging'
    cfSpace: 'Build/Test'
    cfServiceInstance: 'ATI_PERSIEL'
    cfService: 'abap-oem-canary'
    cfServicePlan: 'standard'
    abapSystemAdminEmail: 'xyz@sap.com'
    abapSystemDescription: 'Partner SaaS on ABAP (Production)'
    abapSystemIsDevelopmentAllowed: false
    abapSystemID: 'ATI'
    abapSystemSizeOfPersistence: 4
    abapSystemSizeOfRuntime: 1
    abapSystemParentSaasAppname: 'asp-itapc-canary'
    includeAddon: true
    confirmDeletion: true
```

Reply:
-> In regards to this specific file "abapEnvironmentCreateSystem.md": there is no general section in this example. Therefore, it needs to be added here.

Reply:
Can you also give more information on that @peterpersiel ?

Reply:
> Can you also give more information on that @peterpersiel ?

There is no concept available yet for abapSystemParentService.. (in comparison to SaaS solution on abap-oem). But still the parameters are defined as (empty) default parameters while the service is created.

Regarding the add-on, I am not aware of any differences depending on if abapSystemParentService.. or abapSystemParentSaasAppname is used.

Reply:
Ok. I will leave all parameters as in the current example and added `includeAddon`, `confirmDeletion` and `addonDescriptorFileName` example values. Do you agree on that?

Reply:
@dominiklendle Sounds good to me